Objects/longobject.c

```diff
@@ -6681,3 +6681,67 @@ Py_ssize_t
 PyUnstable_Long_CompactValue(const PyLongObject* op) {
     return _PyLong_CompactValue(op);
 }
+
+PyObject* PyLong_FromInt32(int32_t value)
+{ return PyLong_FromNativeBytes(&value, sizeof(value), -1); }
+
+PyObject* PyLong_FromUInt32(uint32_t value)
+{ return PyLong_FromUnsignedNativeBytes(&value, sizeof(value), -1); }
+
+PyObject* PyLong_FromInt64(int64_t value)
+{ return PyLong_FromNativeBytes(&value, sizeof(value), -1); }
+
+PyObject* PyLong_FromUInt64(uint64_t value)
+{ return PyLong_FromUnsignedNativeBytes(&value, sizeof(value), -1); }
+
+#define LONG_TO_INT(obj, value, type_name) \
+    do { \
+        int flags = Py_ASNATIVEBYTES_NATIVE_ENDIAN; \
```

Comment:
```suggestion
		if (value == NULL) { \
			return -1; \
		} \
        int flags = Py_ASNATIVEBYTES_NATIVE_ENDIAN; \
```

I think that without this check, `PyLong_AsNativeBytes(obj, NULL, ...)` would segfault.

(Maybe this could be something to correct at the level of `PyLong_AsNativeBytes` though...)

Reply:
I added "value must not be NULL" in the doc.